docs/api/datamodel.rst

```diff
@@ -384,14 +389,58 @@ GCODE analysis information
      - 0..1
      - Object
      - The estimated usage of filament
-   * - ``filament.length``
+   * - ``tool{n}.filament.length``
```

Comment:
Shouldn't this rather be ``filament.tool{n}.length``? Same with ``filament.tool{n}.volume`` below.